src/lib.rs

```diff
@@ -37,43 +37,44 @@ fn iterate_tokens(tokens: TokenStream) -> TokenStream {
 
 fn parse_token(token: TokenTree) -> TokenTree {
 	replace_keywords!(token {
-		Me_Irl => Self,
-		ass => as,
+		//! This made me cry to do -sticks
+		me_iww => Self,
+		boipussy  => as,
```

Comment:
Are you absolutely sure this follows [GitHub's guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines#sexually-obscene-content)? I understand this whole project is a joke, but I don't want any of us breaking any rules here.